src/tasks/types.ts

```diff
@@ -7,7 +7,11 @@ export type Task = {
   jira_id: number | null;
   sort_order: number;
   ultimate_mother_id: number;
-  is_done: boolean;
+  /**
+   * @deprecated Use `status` instead.
```

Comment:
It's not like we have JSDoc to make use of this annotation, but it's not like I have any better ideas either :shrug: 

FYI @tomi-timetac